dags/openshift_nightlies/dag.py

```diff
@@ -1,3 +1,4 @@
+from sqlite3 import connect
```

Comment:
is this being used anywhere? I cant see any reference to it

Reply:
Nope, will clean it up

Reply:
Updated.